src/styles/pages/search.scss

```diff
@@ -0,0 +1,3 @@
+// Sections
```

Comment:
On the first load of [the search page](https://preview-480--base.preview.vtex.app/s/?q=mouse&sort=score_desc&page=0), there are some styles missing:
<img src="https://user-images.githubusercontent.com/381395/163294924-c7ea3f52-6d2b-49b7-8c54-3f55afd38aaf.png" width="50%">

Reply:
Tested now and it's been resolved!